### PR TITLE
Removing unused method arguments

### PR DESCRIPTION
**Pull request by john-science**
## Description

@keckler  recently opened [a ticket](https://github.com/terrapower/armi/issues/732) for an unused method argument.  Well, we fixed his method, but I thought I should do a quick cull through the rest of the repo to see if I could find other methods with unused arguments, and fix those as well.

---

## Checklist

- [X] The code is understandable and maintainable to people beyond the author.
- [X] Tests have been added/updated to verify that the new or changed code works.
- [X] There is no commented out code in this PR.
- [X] The commit message follows [good practices](https://terrapower.github.io/armi/developer/tooling.html).
- [X] All docstrings are still up-to-date with these changes.

### CODE REVIEW

**Review by opotowsky — APPROVED** [review]
The work of a true hero -- good cleanup!

File: armi/reactor/tests/test_components.py [hunk 1]
```diff
@@ -938,12 +938,12 @@ class TestHelix(TestShapedComponent):
         "id": 0.1,
     }
 
-    def test_getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
+    def test_getBoundingCircleOuterDiameter(self):
```
Comment (keckler) [inline armi/reactor/tests/test_components.py:941]: Good catch.

This went through the review of #747 yesterday without any notice!

File: armi/utils/pathTools.py [hunk 2]
```diff
@@ -116,25 +116,10 @@ def isAccessible(path):
...
-    Raises
-    ------
-    ValueError:
-        If the path does not exist or
-
-
-    """
-    pass
-
-
```
Comment (keckler) [inline armi/utils/pathTools.py:132]: This function is just egregious. How did this get in here??

**Review by keckler — APPROVED** [review]
Awesome!

File: armi/utils/pathTools.py (hunk 2 above)
Comment (john-science) [reply armi/utils/pathTools.py:132]: I can't tell. It had been there for three years.  

I know clean up PRs aren't sexy, but man, sometimes they are necessary.

**Comment by ntouran** [comment]
Running internal in-use tests suggests that at least some of these are indeed used by internal plugins. Did yall run the internal tests?

**Pull request merged**